cmd/docker/docker.go

```diff
@@ -55,7 +55,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	cli.SetupRootCommand(cmd)
 
 	flags = cmd.Flags()
-	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
+	flags.BoolVarP(&opts.Version, "version", "v", false, "Show version information")
```

Comment:
I found `docker --version SOME_SUBCOMMAND` executes the subcommand rather than just prints version and quits, but this behavior should be bug.